frame/composable-traits/src/dex.rs

```diff
@@ -191,3 +178,11 @@ pub struct PoolInfo<AccountId> {
 	/// Amount of the admin fee pool charges for the exchange
 	pub admin_fee: Permill,
 }
+
+#[derive(Encode, Decode, TypeInfo, Clone, Default, PartialEq, Eq, Debug)]
+pub struct ConstantProductPoolInfo<AccountId> {
```

Comment:
it is more of Configuraiton kind of. Info may include current exchnage rate.

Reply:
Do you want to rename it to Constant..Configuration ?

Reply:
So @dzmitry-lahoda you prefer it to be renamed as `ConstantProductConfig` ?

Reply:
@hussein-aitlahcen any preferene?